project/Dependencies.scala

```diff
@@ -25,6 +25,7 @@ object Dependencies {
     val jacksonDF = "2.12.2"
     val fastutil = "8.5.4"
     val apkparser = "2.6.10"
+    val typesafeconfig = "1.4.2"
```

Comment:
Isn't this already a dependency via the libraryDependencies? Should take care not to have two different versions of the same dependency in the end

Reply:
I added it early in the development stage, because it did need it to compile.
It appears like I added something in between, and it now works fine without explicitly adding it to its dependencies.
I removed the dependency again.